gtsfm/utils/images.py

```diff
@@ -122,3 +123,32 @@ def match_image_widths(
     scaled_image_i2 = resize_image(image_i2, new_height, new_width)
 
     return scaled_image_i1, scaled_image_i2, scale_factor_i1, scale_factor_i2
+
+
+def get_average_point_color(track: SfmTrack, images: List[Image]) -> Tuple[int, int, int]:
+    """
+    Args:
+        track: 3d point/landmark and its corresponding 2d measurements in various cameras
+        images: list of all images for this scene
+
+    Returns:
+        r: red color intensity, in range [0,255]
+        g: red color intensity, in range [0,255]
+        b: red color intensity, in range [0,255]
+    """
+    rgb_measurements = []
+    for k in range(track.number_measurements()):
+
+        # process each measurement
+        i, uv_measured = track.measurement(k)
+        img_h, img_w, _ = images[i].value_array.shape
```

Comment:
you should use `height` and `width` property of image directly.

Reply:
thanks, i'll fix it

Reply:
fixed